libarchive/PKGBUILD

```diff
@@ -24,46 +24,34 @@ prepare() {
   # msysize patch
   #patch -Np1 -i "${srcdir}/libarchive-3.3.1-msys2.patch"
   patch -Np1 -i "${srcdir}/libarchive-3.3.2-bcrypt-fix.patch"
-  autoreconf -ivf
 }
 
 build() {
-  export lt_cv_deplibs_check_method='pass_all'
-  cd "${pkgname}-${pkgver}"
+  local build_type=Release
```

Comment:
local custom variables should start with underscore `_` Link https://man.archlinux.org/man/PKGBUILD.5

Reply:
hm, why?

Reply:
"This will prevent any possible name clashes with internal makepkg variables."

Reply:
yes, but we are in a function scope here, so that shouldn't apply.

Reply:
👍